subworkflows/local/alignment.nf

```diff
@@ -78,6 +80,8 @@ workflow ALIGNMENT {
         bam_ds_bai     = ch_bam_bai_out
         gene_counts    = STAR_ALIGN.out.read_per_gene_tab
         spl_junc       = STAR_ALIGN.out.spl_junc_tab
+        star_samp_col  = ch_sample_collect
```

Comment:
```suggestion
        star_samples  = ch_sample_collect
```
isn't this mainly a channel with star sample metas? I would look into sending `STAR_ALIGN.out.read_per_gene_tab` channel into the analyse transcripts subworkflow and do the processing there.

Reply:
That's true, it will come in on the next PR